modin/core/io/file_dispatcher.py

```diff
@@ -216,7 +216,7 @@ def get_path(cls, file_path):
         if isinstance(file_path, str) and S3_ADDRESS_REGEX.search(file_path):
             return file_path
         else:
-            return os.path.abspath(file_path)
+            return os.path.abspath(os.path.expanduser(file_path))
```

Comment:
What's the general rule on trying to support something that `pandas` currently doesn't support? For example, if you pass in a path like `$HOME/blah/blahhhh/` that isn't automatically expanded in pandas, but is something that we could try to handle.

Reply:
We generally use `modin.experimental` namespace for that. That said, on Windows `$HOME` is not expanded to anything, and I'm not sure where to draw the line at variables expansion on *nix, either - should we then try to support bash-ims like `$(date)/foo/a.csv`?

Reply:
Fair enough!